addons/apcupsd/config.go

```diff
@@ -0,0 +1,136 @@
+package apcupsd
+
+import (
+	"github.com/rocket-pool/smartnode/shared/types/config"
+)
+
+// Constants
+const (
```

Comment:
These docker images are provided by third parties.
Please let me know if in your estimation there is a security risk here and if these images need to be audited.